packages/components/src/local/route/index.spec.tsx

```diff
@@ -0,0 +1,97 @@
+/* global it expect */
+import React from 'react'
+import { mount } from 'enzyme'
+
+import Mapping from '../mapping'
+import { Phase } from '@serge/config'
+import { Route } from './'
+
+const bounds = {
+  imageTop: 14.194809302,
+  imageLeft: 42.3558566271,
+  imageRight: 43.7417816271,
+  imageBottom: 12.401259302
+}
+
+const LocalTileLayer = {
+  url: '/tiles/{z}/{x}/{y}.png',
+  attribution: 'Generated by QTiles'
+}
+
+const history: any = []
+
+const plannedTurns: any = [
```

Comment:
Leave this here for now but we definitely need to start thinking about how our tests are running for the mapping stuff. At the very least we should be using external mocking data. I'll probably look into this next week before it starts to get out of hand.

Reply:
now using external mock data